src/PasswordField.js

```diff
@@ -20,15 +20,12 @@ const getStyles = (props, context, state) => {
 
   const styles = {
     root: {
-      display: 'flex',
-      width: props.fullWidth ? '100%' : null
-    },
-    inputContainer: {
-      flex: 1
+      position: 'relative',
+      display: props.fullWidth ? 'block' : 'inline-block'
     },
     input: {
-      width: '100%',
-      paddingRight: 56
+      paddingRight: 56,
+      boxSizing: 'border-box'
```

Comment:
IE10 [doesn't support `border-box`](http://caniuse.com/#feat=css3-boxsizing). I don't care for IE, but you seem to do, so does it still work there?